modules/features/falcon_feature_content_blocks/config/install/field.field.paragraph.hero_image_heading_subheading.field_image.yml

```diff
@@ -0,0 +1,37 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_image
+    - paragraphs.paragraphs_type.hero_image_heading_subheading
+  module:
+    - image
+id: paragraph.hero_image_heading_subheading.field_image
+field_name: field_image
+entity_type: paragraph
+bundle: hero_image_heading_subheading
+label: Image
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  file_directory: 'paragraphs/hero_image_heading_subheading/[date:custom:Y]-[date:custom:m]'
```

Comment:
I think would be nice to keep this format as a standard. `entity_type/entity_bundle/date`